spec/collectionspace/converter/publicart/acquisition_spec.rb

```diff
@@ -40,69 +40,44 @@
     end
 
     context 'fields overridden by publicart' do
-      context 'when local place' do
+      context 'authority/vocab fields' do
         [
-          "/document/#{p}/owner",
-          "/document/#{p}/acquisitionSource",
+          "/document/#{p}/owners/owner",
+          "/document/#{p}/acquisitionSources/acquisitionSource",
+          "/document/#{p}/acquisitionMethod",
           "/document/#{p}/acquisitionFundingList/acquisitionFunding/acquisitionFundingCurrency",
           "/document/#{p}/acquisitionFundingList/acquisitionFunding/acquisitionFundingSource"
         ].each do |xpath|
           context "#{xpath}" do
-            it 'all values will be URNs' do
```

Comment:
@Paulmulonzia - a lot of these tests were passing when they shouldn't have due to a bug in my test pattern that you were following. I'd changed the `urn_values` method to output all lowercase text, but the test was still complaining only if the result contained capital letters. So, essentially, the test could not fail. I fixed this in the tests. Then I fixed the code that caused the fixed tests to fail.